example/src/computeSelectedTriangles.js

```diff
@@ -95,32 +94,14 @@ export function computeSelectedTriangles( mesh, camera, selectionTool, params )
 			const hull = getConvexHull( projectedBoxPoints );
 			const hullSegments = connectPointsWithLines( hull, boxLines );
 
-			// If a lasso point is inside the hull then the box cannot be contained inside the lasso, so it must be intersected by the lasso.
+			// If any lasso point is inside the hull (arbitrarily checking the first) then the bounds are intersected by the lasso.
 			if ( isPointInsidePolygon( segmentsToCheck[ 0 ].start, hullSegments ) ) {
 
 				return INTERSECTED;
 
 			}
 
-			// determine if the box is intersected by the lasso by counting the number of crossings
-			// https://en.wikipedia.org/wiki/Point_in_polygon#Ray_casting_algorithm
-			const firstPointCrossings = pointRayCrossesSegments(
-				hull[ 0 ],
-				segmentsToCheck
-			);
-			if (
-				hull.some(
-					( point ) =>
-						pointRayCrossesSegments( point, segmentsToCheck ) !==
-						firstPointCrossings
-				)
-			) {
-
-				return INTERSECTED;
-
-			}
```

Comment:
It looks like this PR has boiled down to just removing these point-crossing line tests since it can result in false reports of lasso intersection with the hull, correct? Have you observed any performance differences from removing this?

Reply:
Yes, that's all there is to this PR. I tried increasing the number of segments in the torus and comparing performance, but didn't get any noticeable differences unfortunately, so hopefully, if nothing else, this PR serves to make the algorithm simpler